docs/Changelog.md

```diff
@@ -29059,6 +29059,127 @@ This version of the operator has been available since version 23 of the default
 <dd>Constrain input and output types to all tensor types.</dd>
 </dl>
 
+### <a name="RotaryEmbedding-23"></a>**RotaryEmbedding-23**</a>
```

Comment:
Do we plan to also update the ONNX exporter to automatically export to this op for relevant HF models? 
Have we done a survey of which of the models this RotaryEmbedding op can be directly applied to? I know some of them also use trained positional embeddings.

Reply:
In addition, the llama models use a complex number implementation of RoPE. Do you know if this op proposal can cover that case?

Reply:
Hi @yuanyao-nv : do you have any simple example of the llama complex number RoPE? My guess is that that would likely be an issue for the exporter to deal with. Conceptually, the proposed implementation should cover it, in the sense that RoPE is dependent on interpreting a real-valued embedding of size N as (N/2) planar coordinates, which is rotated by a specified angle. It is just about interpreting the input as either N reals or N/2 planar coordinates, which can be represented as a complex number too. (Similar issues arise for FFT ops.) So far, the preference is to use real-values in ONNX, so it is a matter of mapping pytorch's complex-type to onnx's real-type in the exporter.